django_practice_2/artists/views.py

```diff
@@ -39,8 +51,10 @@ def artist(request, artist_id):
             the DB. Then render the 'artist.html' template sending the 'artist'
             object as context
     """
-    pass
-
+    try:
+        artist = Artist.objects.get(id=artist_id)
```

Comment:
There's a cool one-line helper for this logic:
https://docs.djangoproject.com/es/2.1/topics/http/shortcuts/#get-object-or-404